.github/CHANGELOG.md

```diff
@@ -84,45 +137,110 @@
       return qml.density_matrix(wires=[1])  # wire 0 is traced out
   ```
 
-* The controlled rotation operations `CRX`, `CRY`, `CRZ`, and `CRot`, now have
-  parameter-shift gradients defined, allowing for analytic gradients on hardware
-  without decomposition.
-  [(#915)](https://github.com/PennyLaneAI/pennylane/pull/915)
-  
+* Adds the square root X gate `SX`. [(#871)](https://github.com/PennyLaneAI/pennylane/pull/871)
+
+  ```python
+  dev = qml.device("default.qubit", wires=1)
+
+  @qml.qnode(dev)
+  def circuit():
+      qml.SX(wires=[0])
+      return qml.expval(qml.PauliZ(wires=[0]))
+  ```
+
+* Two new hardware-efficient particle-conserving templates have been implemented
+  to perform VQE-based quantum chemistry simulations. The new templates apply
+  several layers of the particle-conserving entanglers proposed in Figs. 2a and 2b
+  of the article by Barkoutsos *et al*. in [arXiv:1805.04340](https://arxiv.org/abs/1805.04340)
+  [(#875)](https://github.com/PennyLaneAI/pennylane/pull/875)
+  [(#876)](https://github.com/PennyLaneAI/pennylane/pull/876)
+
+<h4>Estimate and track resources</h4>
+
+* The `QuantumTape` class now contains basic resource estimation functionality. The method
+  `tape.get_resources()` returns a dictionary with a list of the constituent operations and the
+  number of times they were run. Similarly, `tape.get_depth()` computes the circuit depth.
+  [(#862)](https://github.com/PennyLaneAI/pennylane/pull/862)
+
+  ```pycon
+  >>> with qml.tape.QuantumTape() as tape:
+  ...    qml.Hadamard(wires=0)
+  ...    qml.RZ(0.26, wires=1)
+  ...    qml.CNOT(wires=[1, 0])
+  ...    qml.Rot(1.8, -2.7, 0.2, wires=0)
+  ...    qml.Hadamard(wires=1)
+  ...    qml.CNOT(wires=[0, 1])
+  ...    qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))
+  >>> tape.get_resources()
+  {'Hadamard': 2, 'RZ': 1, 'CNOT': 2, 'Rot': 1}
+  >>> tape.get_depth()
+  4
+  ```
+
+* The number of device executions over a QNode's lifetime can now be returned using `num_executions`.
+  [(#853)](https://github.com/PennyLaneAI/pennylane/pull/853)
+
+  ```pycon
+  >>> dev = qml.device("default.qubit", wires=2)
+  >>> @qml.qnode(dev)
+  ... def circuit(x, y):
+  ...    qml.RX(x, wires=[0])
+  ...    qml.RY(y, wires=[1])
+  ...    qml.CNOT(wires=[0, 1])
+  ...    return qml.expval(qml.PauliZ(0) @ qml.PauliX(1))
+  >>> for _ in range(10):
+  ...    circuit(0.432, 0.12)
+  >>> print(dev.num_executions)
+  10
+  ```
+
 <h3>Improvements</h3>
```

Comment:
I tried to order the improvements section from most useful to users (improvements that are user-facing or even verging on new user-facing features), down to least useful (improvements that are implementation details).